eng/pipelines/runtime-sanitized.yml

```diff
@@ -81,6 +82,38 @@ extends:
                   scenarios:
                     - normal
 
+      #
+      # Build the whole product with CoreCLR and run libraries tests with AddressSanitizer
+      #
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          buildConfig: Debug
```

Comment:
ASAN is supported in release, but it can miss some checks as the instrumentation pass runs after many optimization passes, which can cause false negatives on some of the stack usage validation.